english/Raiders/LetsgoRaiders.md

```diff
@@ -1,3 +1,6 @@
 Lets go Raiders!
 
-[Here's the link](https://www.google.com)
\ No newline at end of file
+[Here's the link](https://www.google.com)
+
+You see Batman at the Raiders game you are attending and you decide
+to go ask him some personal questions. [This is what you find](https://github.com/udacity/create-your-own-adventure/blob/master/english/batman/batman.md)!
```

Comment:
This link seems to be an absolute link to GitHub rather than a relative link.  If you are linking to another page in the create-your-own-adventure repository, please use relative links to do so.  See the README of this repository for more information.